### PR TITLE
add Security team

### LINKED ISSUE #716: Add Security team
It would be valuable to give folks working on [security across Jupyter](https://github.com/jupyter/security) access to advisories on the JupyterHub org, to not rely on maintainers for creating the drafts in the first place, and ensure security folks are available in the advisory discussions without needing to add collaborators one-by-one.

Proposal:

add a 'security' team, and grant them 'Security Manager' permissions.

Tasks:

- [x] document security team's permissions, membership, and how membership is managed
- [x] create security team
- [x] grant security manager role to the team
- [x] add some members (who? Perhaps the whole jupyter/security Council, plus some members of the working group, as selected by the Council)

More info:

- [security manager documentation](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/managing-security-managers-in-your-organization)
- security manager role is assigned [here](https://github.com/organizations/jupyterhub/settings/security_analysis)

cc @rpwagner @dlqqq

### LINKED ISSUE #720: Add @dlqqq to security team
We're discussing creating a security team here:

- #716 

Part of the motivation for this is so that we can add members to the team from other parts of Jupyter, to facilitate security reporting and coordination across the org.

@dlqqq is one-such person, and has requested membership in this team if #716 is created. So this issue is to track answering the question:

**Should we add @dlqqq to the security team (if it is created)**

We've discussed this a bit via our listserv and there haven't been objections, and we agreed to make the issue first so there was a public record.

### Actions

- [ ] Decide on #716
- [ ] If accepted, decide to add @dlqqq
- [ ] Implement the decision

### PR DESCRIPTION
**Pull request by minrk**
document security role and add @rpwagner, @dlqqq. @rpwagner can you confirm both of these memberships, and can both of you confirm the info in the membership list?

I've created the @jupyterhub/security team with Security Manager permissions, what remains is to add members.

closes #720
closes #716

### CODE REVIEW

**Review by willingc — APPROVED** [review]
Thanks @minrk

**Review by dlqqq — APPROVED** [review]
I can confirm that I would like to have my account added to the security manager role. The messaging proposed here looks good to me.

File: docs/team/structure.md
Comment (yuvipanda) [inline docs/team/structure.md:192]:
```suggestion
```

**Review by yuvipanda — APPROVED** [review]

**Comment by willingc** [comment]
After @rpwagner reviews, let's go ahead and merge this PR. Thanks all.

**Review by choldgraf — APPROVED** [review]
This looks good to me, thanks for writing this up, it's helpful to see it documented like this.

**Review by manics — APPROVED** [review]

**Review by rpwagner — APPROVED** [review]
Confirming my access, thanks for documenting this process. It will help as more subprojects add this role.

**Comment by minrk** [comment]
Thanks all! I've added @rpwagner and @dlqqq to the security team with Security Manager permissions

**Pull request merged**
